beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -270,23 +281,22 @@ func (vs *Server) getPayloadHeaderFromBuilder(
 		return nil, errors.Wrap(err, "could not validate builder signature")
 	}
 
-	maxBlobsPerBlock := params.BeaconConfig().MaxBlobsPerBlock(slot)
 	var kzgCommitments [][]byte
 	if bid.Version() >= version.Deneb {
-		kzgCommitments, err = bid.BlobKzgCommitments()
-		if err != nil {
-			return nil, errors.Wrap(err, "could not get blob kzg commitments")
+		dBid, ok := bid.(builder.BidDeneb)
+		if !ok {
+			return nil, errors.New("builder returned non-deneb or above bid")
```

Comment:
```suggestion
			return fmt.Errorf("bid type %T does not implement builder.BidDeneb", dBid)
```
For unity with above warning logs